packages/twenty-server/src/engine/core-modules/outbound-message-domain/drivers/aws-ses/services/aws-ses-driver.service.ts

```diff
@@ -0,0 +1,258 @@
+import { Logger } from '@nestjs/common';
+
+import {
+  CreateEmailIdentityCommand,
+  CreateTenantCommand,
+  CreateTenantResourceAssociationCommand,
+  GetEmailIdentityCommand,
+  PutEmailIdentityDkimAttributesCommand,
+} from '@aws-sdk/client-sesv2';
+
+import { type AwsSesDriverConfig } from 'src/engine/core-modules/outbound-message-domain/drivers/interfaces/driver-config.interface';
+import { type OutboundMessageDomainDriverInterface } from 'src/engine/core-modules/outbound-message-domain/drivers/interfaces/outbound-message-domain-driver.interface';
+
+import { type AwsSesClientProvider } from 'src/engine/core-modules/outbound-message-domain/drivers/aws-ses/providers/aws-ses-client.provider';
+import { type AwsSesHandleErrorService } from 'src/engine/core-modules/outbound-message-domain/drivers/aws-ses/services/aws-ses-handle-error.service';
+import { OutboundMessageDomainStatus } from 'src/engine/core-modules/outbound-message-domain/drivers/types/outbound-message-domain';
+import { type VerificationRecord } from 'src/engine/core-modules/outbound-message-domain/drivers/types/verifications-record';
+import { type OutboundMessageDomain } from 'src/engine/core-modules/outbound-message-domain/outbound-message-domain.entity';
+
+export class AwsSesDriver implements OutboundMessageDomainDriverInterface {
+  private readonly logger = new Logger(AwsSesDriver.name);
+
+  constructor(
+    private readonly config: AwsSesDriverConfig,
+    private readonly awsSesClientProvider: AwsSesClientProvider,
+    private readonly awsSesHandleErrorService: AwsSesHandleErrorService,
+  ) {}
+
+  async verifyDomain(
+    domain: OutboundMessageDomain,
+  ): Promise<OutboundMessageDomain> {
+    try {
+      this.logger.log(`Starting domain verification for: ${domain.domain}`);
```

Comment:
We should rather use the metricService or auditService 
(metrics is more for grafana alerting if volume changes suddenly and auditService is more complete trace to understand what happen, but we never log to the server - so check if you left any other log)